remote_test_run/run_transfer_a.py

```
@@ -0,0 +1,7 @@
+import asyncio, os, sys
+sys.path.insert(0, "/Users/martino/repos/file-fridge")
+os.environ["DATABASE_PATH"] = "/Users/martino/repos/file-fridge/remote_test_run/instance_a/data/file_fridge_a.db"
+os.environ["SECRET_KEY"] = "dummy_key_for_remote_testing"
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to: "SECRET_KEY"

(S105)